src/nvim/option.c

```diff
@@ -2440,11 +2440,9 @@ did_set_string_option (
     if (briopt_check(curwin) == FAIL)
       errmsg = e_invarg;
   }
```

Comment:
It's not auto-formatting if it's done manually :P 

Still though, there's an `if` directly after it, and you don't want to break up the conditions by the comment, so I'd still prefer the first suggestion, even if the comment is technically in the wrong block.